compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/Names.java

```diff
@@ -139,8 +139,8 @@
 
     // Names related to streams
     public static final Name CONSTRUCT_STREAM = new Name("construct");
-    public static final Name ABSTRACT_ITERATOR = new Name("_Iterator");
-    public static final Name ABSTRACT_CLOSEABLE_ITERATOR = new Name("_CloseableIterator");
+    public static final Name ABSTRACT_ITERATOR = new Name("_StreamImplementor");
+    public static final Name ABSTRACT_CLOSEABLE_ITERATOR = new Name("_CloseableStreamImplementor");
```

Comment:
Shall we rename these to include `STREAM` in the names?

Reply:
Updated in bbcea7f314bd9439481336ba5d07595540458947